var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -389,19 +389,27 @@ def common_config_args(self):
         else:
             # FIXME: those could work for other versions
             png = self.spec['libpng']
+            config_args.append('-system-libpng')
+            if not png.external:
+                config_args.extend([
+                    '{0}'.format(png.libs.search_flags),
```

Comment:
```suggestion
                    png.libs.search_flags,
```
No need to convert to a string, it's already a string.